main.py

```diff
@@ -1,7 +1,38 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Drama"]
```

Comment:
When you have a determined container that does not expect to expand or decline in size or change its content, a tuple is a more efficient data structure for it than a list.